src/sentry/api/event_search.py

```diff
@@ -664,8 +664,20 @@ def visit_numeric_in_filter(self, node, children):
         return self._handle_basic_filter(search_key, operator, search_value)
 
     def visit_numeric_filter(self, node, children):
-        (search_key, _, operator, search_value) = children
-        return self._handle_numeric_filter(search_key, operator, search_value)
+        (negation, search_key, _, operator, search_value) = children
+        if isinstance(operator, Node):
+            operator = handle_negation(negation, operator)
+        elif isinstance(operator, list):
+            operator = operator[0]
```

Comment:
When is `operator` a `Node` or a `list`? 🤔 
I noticed I'm not handling the negation here and it's not clear to me if when this is a list, the operator gets passed down to `_handle_numeric_filter`

Reply:
This might be an artifact from when we used to call `visit_numeric_filter` directly. I think if removing this doesn't break any tests its likely safe to omit

Reply:
It broke some tests so I'm just going to leave it in 😅